controllers/rhmi/rhmi_controller.go

```diff
@@ -781,7 +781,7 @@ func (r *RHMIReconciler) preflightChecks(installation *rhmiv1alpha1.RHMI, instal
 			},
 		}
 		if exists, err := k8s.Exists(context.TODO(), r.Client, secret); err != nil {
-			return ctrl.Result{}, err
+			return result, err
```

Comment:
Just a side note.
This isn't really necessary. Reconcilers do re-trigger when:
- there's an error 
- when you tell them to in result{requeue}
Since in here, error is always there, the requeue shouldn't be happening because we are going to reconcile again due to error.

Reply:
Good point! I update both occurrences back to `ctrl.Result{}, err` since that makes more sense, thanks for explanation